keyboards/sofle/keyhive/readme.md

```diff
@@ -1,66 +1,53 @@
--	OLED displays current layer, lock key status, WPM, custom logo, and custom name.
--	OLED shuts off on idle and when computer is sleeping.
--	Includes working Luna keyboard pet.
--	Includes working Bongocat (disable RGB for sufficient space).
--   Includes stock OLED code as well.
-
-### RGB Features:
--   Adds custom layer lighting with custom gradients for each layer.
--   RGB underglow support and remapped to physical locations (nicer gradients and effects).
--   Adds white caps lock, scroll lock, and num lock key indicators using the top row/underglow LEDs.
+### Acknowledgements
 
-### Rotary Encoder and VIA Features:
--   Fully emappable left encoder controls volume up/down/mute. Right encoder PGUP/PGDOWN.
--   Allows for  live remapping of per-layer rotary encoder functions in VIA.
--   VIA support included by default.
--   This fixes the Keyhive left bottom row offset issue in VIA. You will need to import sofle.json in VIA.
--   Custom macro key in VIA for Super Alt Tab, which is fully compatible with rotary encoders.
--   Custom macro key in VIA for moving windows to other monitors in Windows, which is fully compatible with rotary encoders.
+* Solartempest - the image on this page and most of the code is either copied directly or inspired by their fork. [Solartempest's fork.](https://github.com/solartempest/qmk_firmware/tree/master/keyboards/solartempest/sofle).
+* [Keyhive fork](https://github.com/keyhive/qmk_firmware) defined all of the board settings.
 
-### Tap and Other Features:
--   Push left-shift + backspace to delete whole words. Right-shift + backspace to delete whole words in the opposite direction.
--   Double tap layers to stay on a layer instead of momentary push.
--   Adds key combo functionality.
--   Symmetric modifiers (CMD/Super, Alt/Opt, Ctrl, Shift).
+# Supported Keymaps
+* [keyhive_via](../keymaps/keyhive_via/readme.md) - Includes special support for remapping encoders with VIA.
+* Backwards compatibility for other keymaps is included but has not been tested.
 
-## Using with VIA
+# VIA Support
+As of 1.3.1, the VIA tool does not support Keyhive/Sofle V2 out of the box.
+See [keyhive_via](../keymaps/keyhive_via/readme.md) for details about configuring and using VIA.
 
--   After flashing, in VIA make sure to Import Keymap, which is "sofle VIA keymap.json". This will alow VIA to recognize the updated layout and custom functions. VIA will not auto-recognize the keyboard with this firmware because of the necessary customization.
--   Go to Save+Load to Load Saved Layout. You can import my own layout "sofle VIA layout.json" or just use the Keymap tab to assign your own keys. Having another keyboard connected can be handy for doing this step.
--   It is a good idea to Save Current Layout after you decide on your mapping.
+# Compiling
 
+Make example for this keyboard:
+```sh
+qmk compile -kb sofle/keyhive -km keyhive_via
+```
```

Comment:
```suggestion

    make sofle/keyhive:default
```

Reply:
Are `qmk compile` and `qmk flash` deprecated? It was really convenient that you could run those commands from any directory in the repo.

Reply:
No, but the make commands are what's currently in the template. Eventually that will change.

Reply:
Ok, keeping things consistent until a migration sounds like it would make things easier.

Reply:
I just noticed that you changed the keymap here to default. Almost all of my testing was done with the "keyhive_via" keymap, which was tailor-made for this version of the board. So I wanted to use that as the default.

Reply:
Again, per the template the "example commands" should use the `default` keymap. They are, after all, examples, not a full guide on compiling a particular board.

Reply:
Thanks, updated everything to use the default keymap!